frontend/src/container/MetricsApplication/Tabs/External/Table/index.tsx

```diff
@@ -0,0 +1,187 @@
+	const { selectedTime } = useSelector<AppState, GlobalReducer>(
+		(state) => state.globalTime,
+	);
+	const [addressFilter, setAddressFilter] = React.useState<string>('');
+	const [reduceTo, setReduceTo] = React.useState<ReduceTo>('latest');
+
+	const { t } = useTranslation(['common']);
+	const { notifications } = useNotifications();
+	const errorMessage = t('something_went_wrong');
+
+	// BEGIN Request rate
+	const reqRateQuery: GetQueryResultsProps = makeReqRateQuery({
+		widgetId: `${widgetId}-req-rate`,
+		selectedTime,
+		serviceName,
+	});
+
+	const reqRateResponse = useQuery<
+		SuccessResponse<MetricRangePayloadProps> | ErrorResponse
+	>(`${widgetId}-req-rate`, () => GetMetricQueryRange(reqRateQuery));
+
+	if (reqRateResponse.isError) {
+		notifications.error({
+			message: errorMessage,
+			description: `Unable to fetch request rate data. ${reqRateResponse.error?.toString()}`,
+		});
+	}
+	// END Request rate
+
+	// BEGIN Duration
+	const durationQuery: GetQueryResultsProps = makeDurationQuery({
+		widgetId: `${widgetId}-duration`,
+		selectedTime,
+		serviceName,
+	});
+
+	const durationResponse = useQuery<
+		SuccessResponse<MetricRangePayloadProps> | ErrorResponse
+	>(`${widgetId}-duration`, () => GetMetricQueryRange(durationQuery));
+
+	if (durationResponse.isError) {
+		notifications.error({
+			message: errorMessage,
+			description: `Unable to fetch duration response data. ${durationResponse.error?.toString()}`,
+		});
+	}
+	// END Duration
+
+	// BEGIN Error percentage
+	const errPercentQuery: GetQueryResultsProps = makeErrPercentQuery({
+		widgetId: `${widgetId}-err-percent`,
+		selectedTime,
+		serviceName,
+	});
+
+	const errPercentResponse = useQuery<
+		SuccessResponse<MetricRangePayloadProps> | ErrorResponse
+	>(`${widgetId}-err-percent`, () => GetMetricQueryRange(errPercentQuery));
+
+	if (errPercentResponse.isError) {
+		notifications.error({
+			message: errorMessage,
+			description: `Unable to fetch error percentage data. ${errPercentResponse.error?.toString()}`,
+		});
+	}
+	// END Error percentage
+
+	const tableRows = useMemo(
+		() =>
+			makeTableRows({
+				duration: durationResponse.data?.payload?.data.result || [],
+				errPercent: errPercentResponse.data?.payload?.data.result || [],
+				reqRate: reqRateResponse.data?.payload?.data.result || [],
+				reduceTo,
+			}),
+		[
+			reqRateResponse.data,
+			durationResponse.data,
+			errPercentResponse.data,
+			reduceTo,
+		],
+	);
+
+	const tableRowsFiltered = useMemo(() => {
+		if (!addressFilter) {
+			return tableRows;
+		}
+		return tableRows.filter((row) => row.address.includes(addressFilter));
+	}, [tableRows, addressFilter]);
+
+	return (
+		<div>
+			<Toolbar
+				addressFilter={addressFilter}
+				onAddressFilterChange={setAddressFilter}
+				reduceTo={reduceTo}
+				onReduceToChange={setReduceTo}
+			/>
+			<ResizeTable
+				key={addressFilter /* Force cell re-render on filter value change. */}
+				columns={[
```

Comment:
can we also make the columns the same we way we are making the row

Reply:
Fixed